include/envoy/json/json_object.h

```diff
@@ -159,6 +165,21 @@ class Object {
   virtual std::string asString() const PURE;
 
   /**
+   * @return the value of the object as a boolean (where the object is a boolean).
+   */
+  virtual bool asBoolean() const PURE;
```

Comment:
I needed a way to access the contents in tests. It seemed generic a thing to do that I added it to the interface.